server/main.lua

```diff
@@ -1,81 +1,92 @@
 local config = require 'config.server'
 local sharedConfig = require 'config.shared'
-local Bail = {}
+local bail = {}
 local currentTruckers = {}
 
 RegisterNetEvent('qbx_truckerjob:server:doBail', function(bool, vehInfo)
-    local client = source
-    local Player = exports.qbx_core:GetPlayer(client)
-    if not Player then return end
+    local src = source --[[@as number]]
```

Comment:
No reason to localize src here we have the same data inside our player we have localized.

Just do Player.PlayerData.source